### ISSUE TITLE
[Feat] Changes in Layer Manager tabs order and labels

### ISSUE THREAD

**Issue by petrasovaa** [OSGeo/grass#799]
Tab order in GUI: 'Modules' should be 3rd (between 'Layers' and 'Console'). This order will make more sense, in data catalog you select data, you display them, find module to run analysis and check output in console.

Tab 'Layers' could be renamed to 'Display' to highlight the difference between the data layers in catalog and displayed layers.
